requirements/pytorch/extra.txt

```diff
@@ -7,3 +7,4 @@ omegaconf>=2.0.5, <2.3.0
 hydra-core>=1.0.5, <1.3.0
 jsonargparse[signatures]>=4.18.0, <4.19.0
 rich>=10.14.0, !=10.15.0.a, <13.0.0
+tensorboardX>=2.2, <=2.5.1  # min version is set by torch.onnx missing attribute
```

Comment:
Since it's not a base dependency anymore maybe this should be `tensorboard` again

Reply:
yeah why not

Reply:
`tensorboard` is even worse than `tensorboardX` because it depends on `grpcio`, which required compilation on M1/M2 with Python >=3.10, and still does (tested now)
so there may still be a rationale for going with `tensorboardX` for extras